client/plots/singleCellPlot.js

```diff
@@ -536,6 +530,23 @@ class singleCellPlot {
 				chartType: 'singleCellPlot',
 				settingsKey: 'showGrid',
 				title: 'Show grid'
+			},
+			{
+				label: 'Show no expression',
+				boxLabel: '',
+				type: 'checkbox',
+				chartType: 'singleCellPlot',
+				settingsKey: 'showNoExpCells',
+				title: 'Show cells without expression'
+			},
+			{
+				label: 'Max expression percentile',
```

Comment:
please do not add it as config option. instead pass it to ColorScale via `numericInputs{}` so it allows customizing auto/fixed/percentile scales

Reply:
The color scale allows to set a new min and max, not a percentile, do you mean we should support this in the color scale?

Reply:
yes do it as `numericInputs{cutoffMode:'percentile',defaultPercentile:?}` see client/dom/types/colorScale.ts

Reply:
Done